compiler/x/codegen/OMRCodeGenerator.hpp

```diff
@@ -594,7 +594,8 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
 
    bool nodeIsFoldableMemOperand(TR::Node *node, TR::Node *parent, TR_RegisterPressureState *state);
 
-   TR::IA32ConstantDataSnippet     *findOrCreateConstant(TR::Node *, void *c, uint8_t size);
+   TR::IA32DataSnippet         *createData(TR::Node *, void *c, uint8_t size);
```

Comment:
If this is the API for creating an arbitrary size data snippet (the small 4,8,16 byte primitives have a handy wrapper), I wish the name was a little more descriptive than just `createData`.  Even `createDataSnippet` might be better.

Reply:
Good suggestion, I was just to keep the old names. They are updated now.